### ISSUE TITLE
ERROR - set_automated_followed_pool error line contains NULL byte

### ISSUE THREAD

**Issue by ancuse1** [InstaPy/InstaPy#1018]
First of all congrats on the work done, it is awesome all the features InstaPy has!

## Expected Behavior
Load correctly the .csv file in logs folder and unfollow those who are not following my account
## Current Behavior
Error is thrown printing :

> ERROR - set_automated_followed_pool error line contains NULL byte

## Possible Solution (optional)
Looked around on Stackoverflow on this [question](https://stackoverflow.com/questions/7894856/line-contains-null-byte-in-csv-reader-python) and tried opening it with codecs but couldn't make it work.
## InstaPy configuration
Yesterday I pulled the newest version and here is the line where I call the unfollow method from my quickstart.py:
`session.unfollow_users(amount=500, onlyNotFollowMe=True, sleep_delay=60)`

Thanks in advance!

**Comment by omidedipidizi** [comment]
i got this issue on raspberry pi too !!!

**Comment by converge** [comment]
https://github.com/timgrossmann/InstaPy/search?utf8=%E2%9C%93&q=%22set_automated_followed_pool+error%22&type=Issues

**Comment by ancuse1** [comment]
Hi, I checked all other related issues, but I think they do not apply to this case. I have pulled the latest version just couple days before opening this issue, the error does not complain about any selector not being found nor an index being out of range so I thought it is something else.

I've also checked my csv file and has all the people I followed during last couple weeks using InstaPy. I'll try to start a new csv file during next days and see if the problem persists.

I'll post the results here.
